src/scripts/components/dateselector/wv.dateselector.js

```diff
@@ -22,22 +22,24 @@ class dateSelector extends React.Component {
       date: props.date,
       maxDate: props.maxDate,
       minDate: props.minDate,
-      tab: null
+      tab: null,
+      maxZoom: props.maxZoom
     };
   }
   componentWillReceiveProps(props) {
     this.setState({
       date: props.date,
       maxDate: props.maxDate,
-      minDate: props.minDate
+      minDate: props.minDate,
+      maxZoom: props.maxZoom
     });
   }
   blur() {
     this.setState({tab: null});
   }
   nextTab(index) {
     var nextTab;
-    if (index < 3) {
+    if (index < 5) {
```

Comment:
I think this `5` will need to be dynamic, right @ZachTRice ? could be passed as a tabLength prop based on maxZoom.

Reply:
👍

Reply:
I added a `maxTab` variable and set it based on the `maxZoom` state